### DISCUSSION TITLE
Multiple Providers: remote user claim

### DISCUSSION THREAD

**Discussion by MonsieurRz** [OpenIDC/mod_auth_openidc#1150] [Q&A] [answered]
I was using the single provider with OIDCRemoteUserClaim. 
I am now trying to migrate to the multiple providers configuration so far so good but I cannot find a way to set a remote user claim for a specific provider configuration.

Looking at the code it does not look like it is supported but I could be wrong.
Is the remote user claim supported ?

**Comment by zandbelt** [comment, marked as answer]
`OIDCRemoteUserClaim` is not configurable per-provider, it would open up naming collision; you should stick with the default <sub>@<issuer>, or set headers/environment variables with Apache expressions
